app/src/main/java/com/example/myapplication/FirstFragment.java

```diff
@@ -10,6 +10,7 @@
 import android.view.ViewGroup;
 import androidx.annotation.NonNull;
 import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
```

Comment:
This import can now be removed from HomePage.java as well. Please check on Android Studio for the unused ones to remove them